schedule/yast/encryption/lvm_encrypt_separate_boot_aarch64.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: lvm-encrypt-separate-boot
+description: >
+  Same as lvm-full-encrypt, but with separate boot not encrypted partition, only
+  installation to not repeat everything again with small risk.
+vars:
+  UNENCRYPTED_BOOT: 1
+  ENCRYPT: 1
+  FULL_LVM_ENCRYPT: 1
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  suggested_partitioning:
+    - installation/partitioning/new_partitioning_gpt
+  software:
+    - installation/installation_settings/validate_default_target
+  grub:
```

Comment:
looks better how was done [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17075/files#diff-18707e1b2fa5234c91f7bba8340cab7f55b8a5e01eaeba999041e72f17dc62c0R19) as part of the login:

```
first_login:
    - installation/boot_encrypt
    - installation/first_boot
```
could you please apply everywhere in this PR?

Reply:
I have updated all yaml files:
schedule/yast/lvm/lvm+cancel_existing_cryptlvm_aarch64.yaml
schedule/yast/lvm_multipath_encrypted_aarch64.yaml
schedule/yast/encryption/activate_encrypted_volume+force_recompute_aarch64.yaml
schedule/yast/encryption/lvm_full_encrypt_aarch64.yaml
schedule/yast/encryption/cryptlvm_sle_15_aarch64.yaml
schedule/yast/encryption/crypt_no_lvm_aarch64.yaml
schedule/yast/encryption/lvm_encrypt_separate_boot_aarch64.yaml
  first_login:
    - installation/boot_encrypt
    - installation/first_boot